include/mvd/bits/sonata_misc.hpp

```diff
@@ -216,6 +217,14 @@ inline std::vector<std::string> SonataFile::getMtypes(const Range & range) const
     return pop_->getAttribute<std::string>(did_mtypes, select(range, size_));
 }
 
+inline std::vector<std::string> SonataFile::getEmodels(const Range & range) const{
+    auto model_tpl = pop_->getAttribute<std::string>(did_emodel, select(range, size_));
+    for (auto& model : model_tpl) {
+        model = model.substr(4);
```

Comment:
Maybe it would be better to break it based on the ":" delimiter, in case the "hoc" prefix changes at some point?

Reply:
Yeah, it's a safer strategy